packaging/flatpak/com.github.terrarier2111.Leafish.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: com.github.terrarier2111.Leafish
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: leafish
+rename-icon: leafish-logo
+rename-desktop-file: leafish.desktop
+rename-appdata-file: leafish.metainfo.xml
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--device=dri"
+- "--socket=pulseaudio"
```

Comment:
We don't currently have audio yet, maybe remove this for now till we do?

Reply:
```suggestion
```

Reply:
> We don't currently have audio yet, maybe remove this for now till we do?

If you want to, I can remove it.

Reply:
Please do for now, easy enough to add back later. Let's keep permissions only to what's necessary